src/gl_limits.rs

```diff
@@ -4,12 +4,16 @@ use gleam::gl;
 #[cfg_attr(feature="serde_serialization", derive(Serialize, Deserialize))]
 pub struct GLLimits {
     pub max_vertex_attribs: u32,
+    pub max_tex_size: i32,
```

Comment:
Ah, true. I originally had this as unsigned, but changed it because `level`, `width`, and `height` are signed, but at the time of comparison they are guaranteed to be positive as well. I'll make the update.